### DISCUSSION TITLE
Beta 5 - Hotkey to toggle the thumbnails bar stopped working

### DISCUSSION THREAD

**Discussion by GaryAtlan82** [d2phap/ImageGlass#1639] [Q&A] [answered]
Beta 5 - Hotkey to toggle the thumbnails bar stopped working, I had it set like this pre Beta 5:
```
    "MnuToggleThumbnails": [
      "M",
      "Ctrl+M"
    ],
```

Its still there but now nothing happens, still pressing `H` shows the thumbnails bar. Has the `MnuToggleThumbnails` been changed to something else? How can I get the thumbnails bar to show with a custom key now?

Thanks for any help.

**Comment by d2phap** [comment, marked as answer]
Some of the menu are renamed in beta 5 to sync up with the display name, you can use `MnuToggleGallery` for the Gallery (thumbnail bar)

You can find all menu names here, start with `Mnu` prefix:
https://github.com/d2phap/ImageGlass/blob/4f8152ae11cb01bbc3f81aa431870fcd85d8cf2f/v9/ImageGlass/FrmMain.Designer.cs#L37-L141

**Reply by GaryAtlan82** [reply]
I see, thank you for the answer.
